### DISCUSSION TITLE
Hi, I want to stop an active consumer , without deleting the queue ? how can I do that

### DISCUSSION THREAD

**Discussion by ciftcioguzhan** [rabbitmq/rabbitmq-dotnet-client#1122] [Q&A]
Hi, I want to stop an active consumer , without deleting the queue ? how can I do that @rabbitmq-ci

**Comment by lukebakken** [comment]
* How are you declaring your queues?
* What have you tried so far?

**Comment by ciftcioguzhan** [comment]
> * How are you declaring your queues?
> 
> * What have you tried so far?
> 
> 

i tried basic cancel but it didn't work :(

**Reply by lukebakken** [reply]
> How are you declaring your queues?

Please answer _all_ of my questions! Tell me about the queues you are declaring.
